helpers/controllers/submissionController.ts

```diff
@@ -9,14 +9,14 @@ import _ from 'lodash'
 const { User, Submission, Challenge, Lesson } = db
 
 type ArgsCreateSubmission = {
-  lessonId: string
+  lessonId: number
   cliToken: string
   diff: string
-  challengeId: string
+  challengeId: number
 }
```

Comment:
The argument types are auto-generated as well. You remove the `ArgsCreateSubmission` and `ArgsGetSubmissions` types and do the following:

```ts
import {
  MutationCreateSubmissionArgs,
  QuerySubmissionsArgs
} from '../../graphql'

export const createSubmission = async (
  _parent: void,
  args: MutationCreateSubmissionArgs
) => { // ... }

export const submissions = async (
  _parent: void,
  arg: QuerySubmissionsArgs,
  ctx: Context
) => { // ... }
```